src/components/labware/PacbioRunWell.vue

```diff
@@ -196,15 +196,24 @@ const updateUsedAliquotSource = async (barcode) => {
   const { id, type, volume, concentration, insert_size, template_prep_kit_box_barcode } =
     store.tubeContentByBarcode(barcode)
   const source_type = type === 'pools' ? 'Pacbio::Pool' : 'Pacbio::Library'
-  const used_aliquot = createUsedAliquot({
-    source_id: id,
-    source_type,
-    volume,
-    concentration,
-    insert_size,
-    template_prep_kit_box_barcode,
-    barcode,
-  })
+  const used_aliquot = createUsedAliquot(
+    {
+      source_id: id,
+      source_type,
+      concentration,
+      insert_size,
+      template_prep_kit_box_barcode,
+      barcode,
+      volume,
+    },
+    source_type === 'Pacbio::Library'
```

Comment:
Could we do this check above and then set the volume and available_volume based on it? something like.

```
const available_volume = source_type === 'Pacbio::Library' ? getAvailableVolume.. : null

const used_aliquot = createUsedAliquot(
{
...
volume: available_volume || volume,
available_volume
}
```

Reply:
The idea was to contain it within the used_aliquot type, so we don't need to set this in different places around that which will help any modification in future.
But, I think we can use this source_type === 'Pacbio::Library' check also within the type itself, thereby avoiding these checks to be done everytime. Hope it make sense. Please let me know otherwise.

Reply:
OK, yeah I think the logic should be in one place.